Libraries/NewAppScreen/components/Colors.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow
+ * @format
+ */
+
+'use strict';
+
+export default {
+  primary: '#1292B4',
+  white: '#FFF',
+  lighter: '#F3F3F3',
+  light: '#DAE1E7',
+  dark: '#444',
+  black: '#000',
+};
```

Comment:
We are mixing color names (white and black), hierarchy pattern (primary) and color states (lighter, light, dark). Think would be better to use only one pattern to name colors here, it will help to understand the code.

Suggestion:

```js
export default {
  bondiBlue: '#1292B4',
  white: '#FFF',
  whiteSmoke: '#F3F3F3',
  pattensBlue: '#DAE1E7',
  charcoal: '#444',
  black: '#000',
};
```

I've used this [site as a reference](https://www.htmlcsscolor.com/hex/444444)